tanner/tests/test_api_server.py

```diff
@@ -83,7 +83,8 @@ def get_app(self):
             "message": ["f387d46eaeb1454cadf0713a4a55be49", "e85ae767b0bb4b1f91b421b3a28082ef"]}}
         self.serv.api.return_sessions = mock_return_sessions
         request = await self.client.request("GET",
-                                            "/8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?filters=peer_ip:127.0.0.1 start_time:1497890400 user_agent:ngnix")
+                                            "/8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?\
+                                            filters=peer_ip:127.0.0.1 start_time:1497890400 user_agent:ngnix")
```

Comment:
Could you please check why this test if broken?
If you look at the travis log, you get:
```
FAIL: test_api_sessions_request (tanner.tests.test_api_server.TestAPIServer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/aiohttp/test_utils.py", line 418, in new_func
    func(self, *inner_args, **inner_kwargs))
  File "uvloop/loop.pyx", line 1364, in uvloop.loop.Loop.run_until_complete
  File "/opt/python/3.5.5/lib/python3.5/asyncio/futures.py", line 294, in result
    raise self._exception
  File "/opt/python/3.5.5/lib/python3.5/asyncio/tasks.py", line 240, in _step
    result = coro.send(None)
  File "/home/travis/build/mushorg/tanner/tanner/tests/test_api_server.py", line 88, in test_api_sessions_request
    assert request.status == 200
AssertionError: 
-------------------- >> begin captured logging << --------------------
aiohttp.server: ERROR: Error handling request
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/aiohttp/web_protocol.py", line 422, in start
    resp = yield from self._request_handler(request)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/aiohttp/web.py", line 306, in _handle
    resp = yield from handler(request)
  File "/home/travis/build/mushorg/tanner/tanner/api/server.py", line 62, in handle_sessions
    sessions = await self.api.return_sessions(applied_filters)
  File "/home/travis/build/mushorg/tanner/tanner/tests/test_api_server.py", line 77, in mock_return_sessions
    if type(filters) is dict and filters['peer_ip'] == "127.0.0.1" and \
KeyError: 'peer_ip'
aiohttp.access: INFO: 127.0.0.1 - - [25/Mar/2018:12:07:37 +0000] "GET /8fa6aa98-4283-4085-bfb9-a1cd3a9e56e4/sessions?
```

Reply:
In this line when you put the part of request into new line, tabs also considered - that is why test is failed. Use "+" for line splitting instead